net/stubby/Makefile

```diff
@@ -15,8 +15,8 @@ PKG_MAINTAINER:=Jonathan Underwood <jonathan.underwood@gmail.com>
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://github.com/getdnsapi/$(PKG_NAME)
-PKG_SOURCE_VERSION:=58200cadec6371f95e31a7f3735225c5a46ecf75
-PKG_MIRROR_HASH:=28c46f4464cb41cf59264d10da63dc25ece9a1d00b4dfb05a9276594658e5eb9
+PKG_SOURCE_VERSION:=b0d3154af61e1b46a30b56d239dc074273642217
```

Comment:
This can be changed to PKG_SOURCE_VERSION:=v$(PKG_VERSION) as it is tagged properly.

Reply:
Git(hub) tags can be moved, and aren't really reliable as a result. So, I'm not sure this is a great idea.

Reply:
Have done it anyway.